src/promptflow/tests/sdk_cli_test/conftest.py

```diff
@@ -127,3 +148,32 @@ def serving_client_llm_chat(mocker: MockerFixture):
 @pytest.fixture
 def serving_client_python_stream_tools(mocker: MockerFixture):
     return create_client_by_model("python_stream_tools", mocker)
+
+
+@pytest.fixture
+def mock_for_recordings(request: pytest.FixtureRequest, mocker: MockerFixture) -> None:
+    recording_folder: Path = RECORDINGS_TEST_CONFIGS_ROOT / request.cls.__name__
```

Comment:
why do we do recording_folder instead of just recording file? I think we only designed one file per folder?